File: CuriousFreaksCodingSheet/Array/javascript/Findminimumandmaximum.js
// https://www.geeksforgeeks.org/problems/find-minimum-and-maximum-element-in-an-array4428/1
class Solution {
  getMinMax(arr, n) {
    let min = arr[0];
    let max = arr[0];

    for (let i = 1; i < n; i++) {
      if (max < arr[i]) {
        max = arr[i];
      }
      if (min > a[i] && max > a[i]) {
        min = arr[i];
      }
    }
    var ans = [];
    ans.push(max);
    ans.push(min);
  }
}
var prob = new Solution();
prob.getMinMax();

File: CuriousFreaksCodingSheet/Array/java/Findminimumandmaximum.java
package CuriousFreaksCodingSheet.Array.java;
// https://www.geeksforgeeks.org/problems/find-minimum-and-maximum-element-in-an-array4428/1
public class Findminimumandmaximum {
    public static void main(String[] args) {
        long [] arr = {1, 345, 234, 21, 56789};
        getMinMax(arr,arr.length );
    }

    static void getMinMax(long a[], long n)

    {
        long min = a[0];
        long max = a[0];
        for (int i = 1; i < n; i++) {
            if (max < a[i]) {
                max = a[i];

            }
            if (min >a[i] && max > a[i]) {
                min = a[i];
            }
        }
        System.out.println("min :"+min);
        System.out.println("max :" +max);

        // Write your code here
    }
}
